tests/workorder_json.py

```diff
@@ -0,0 +1,212 @@
+		self.json_obj["params"]["responseTimeoutMSecs"] = responseTimeoutMSecs
+
+	def set_payloadFormat(self, payloadFormat):
+		self.json_obj["params"]["payloadFormat"] = payloadFormat
+
+	def set_resultUri(self, resultUri):
+		self.json_obj["params"]["resultUri"] = resultUri
+
+	def set_notifyUri(self, notifyUri):
+		self.json_obj["params"]["notifyUri"] = notifyUri
+
+	def set_workOrderId(self, workOrderId):
+		self.json_obj["params"]["workOrderId"] = workOrderId
+
+	def get_workOrderId(self):
+		return self.json_obj["params"]["workOrderId"]
+
+	def set_workerId(self, workerId):
+		self.json_obj["params"]["workerId"] = workerId
+
+	def set_workloadId(self, workloadId):
+		self.json_obj["params"]["workloadId"] = workloadId
+
+	def set_requesterId(self, requesterId):
+		self.json_obj["params"]["requesterId"] = requesterId
+
+	def set_workerEncryptionKey(self, workerEncryptionKey):
+		self.json_obj["params"]["workerEncryptionKey"] = workerEncryptionKey
+
+	def set_dataEncryptionAlgorithm(self, dataEncryptionAlgorithm):
+		self.json_obj["params"]["dataEncryptionAlgorithm"] = dataEncryptionAlgorithm
+
+	def add_inData(self, dataHash, data, encryptedDataEncryptionKey=None):
+		index = len(self.json_obj["params"]["inData"])
+		self.json_obj["params"]["inData"].append({})
+		self.json_obj["params"]["inData"][index]["index"] = index
+		self.json_obj["params"]["inData"][index]["dataHash"] = dataHash
+		self.json_obj["params"]["inData"][index]["data"] = data
+		if encryptedDataEncryptionKey:
+			self.json_obj["params"]["inData"][index]["encryptedDataEncryptionKey"] = encryptedDataEncryptionKey
+		else:
+			self.json_obj["params"]["inData"][index]["encryptedDataEncryptionKey"] = ""
+		self.json_obj["params"]["inData"][index]["iv"] = ""
+
+	def add_outData(self, dataHash, encryptedDataEncryptionKey=None):
+		index = len(self.json_obj["params"]["outData"])
+		self.json_obj["params"]["outData"].append({})
+		self.json_obj["params"]["outData"][index]["index"] = index
+		self.json_obj["params"]["outData"][index]["dataHash"] = dataHash
+		self.json_obj["params"]["outData"][index]["data"] = ""
+		if encryptedDataEncryptionKey:
+			self.json_obj["params"]["outData"][index]["encryptedDataEncryptionKey"] = encryptedDataEncryptionKey
+		else:
+			self.json_obj["params"]["outData"][index]["encryptedDataEncryptionKey"] = ""
+		self.json_obj["params"]["outData"][index]["iv"] = ""
+
+	# Use these if you want to pass json to WorkOrderJRPCImpl
+	def get_params(self):
+		params_copy = self.json_obj["params"].copy()
+		params_copy.pop("inData")
+		params_copy.pop("outData")
+		return params_copy
+
+	def get_inData(self):
+		return self.json_obj["params"]["inData"]
+
+	def get_outData(self):
+		return self.json_obj["params"]["outData"]
+
+class WorkOrderGetResultJson(WorkOrderJson):
+	def __init__(self, _id, workOrderId):
+		WorkOrderJson.__init__(self, _id)
+		self.set_method("WorkOrderGetResult")
+		self.set_workOrderId(workOrderId)
+
+	def get_workOrderId(self):
```

Comment:
Convention to function name is snakecase. Change the function name to get_workOrderId->get_work_order_id()

Reply:
Done.